submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,272 @@
+# Burning Questions
+
+*by Sir Fendi*
+
+*version 1.1*
+
+(Calculations will be double-checked)
+
+Media:
+
+* Diagrams are included in a separate PowerPoint file. 
+
+* There is a preview image of the table as a PNG.
+
+All feedback is welcome! I am wondering:
+
+- Maybe more images can be added?
+
+- Do you think think the responses cover the questions sufficiently or maybe I have missed some factors to consider?
+
+- Are there any more questions to add?
+
+- Are there any other posts/explorations you have seen in the past that would be cool to mention here?
+
+. . . . . . . .
+
+Factorio has a variety of fuels that can go into your burner devices, ranging from classic and modest wood, up to sci-fi level nuclear fuel. The most popular burner fuel is coal, which is the backbone of early game automation and it runs all the boilers, furnaces, vehicles, and other machines reliably until it inevitably runs low. 
+
+In the mid game, solid fuel is introduced as the first processed fuel, made initially from crude oil. Conveniently, solid fuel is used to later make rocket fuel, which is in turn used to make nuclear fuel. These processed fuels offer noticeable advantages over coal, like having greater energy density both per unit and per stack, and an acceleration bonus when used in vehicles. More curiously, however, the late game also brings the technology to liquify good old coal and convert it into solid fuel. This raised for me the question: Does processing coal into solid fuel get more energy out of it? In other words, if you liquify 10 units of coal, make solid fuel out of the oils and gas that it yields, and then burn the solid fuel, does it provide more energy than the directly burning the coal, even with all that processing? And what if you further processed it into rocket fuel? Similar burning questions followed and snowballed into this general exploration of the energy contents of different burner fuels as well as the energy costs of converting them. 
```

Comment:
> solid fuel is introduced as the first processed fuel, made initially from crude oil.
The word "initially" seems wrong here, cause Solid Fuel can only be produced from Heavy/Light oil or Petroleum gas, (unless you're referring to coal liquefaction, but then you should probably make that clearer, and most people don't switch to it anyways).

As an aside, I just checked on the wiki, and apparently Solid fuel requires the Advanced oil processing research, so changing it to "initially made from Petroleum gas" would also not be completely accurate as the user should immediately be using Light oil to create Solid fuel.